frontend/src/components/workspace/CompetitionFormsTable.tsx

```diff
@@ -0,0 +1,185 @@
+import { ApplicationFormDetail } from "src/types/applicationResponseTypes";
+import { CompetitionForms } from "src/types/competitionsResponseTypes";
+import { FormDetail, FormInstruction } from "src/types/formResponseTypes";
+
+import { useTranslations } from "next-intl";
+import Link from "next/link";
+import { Table } from "@trussworks/react-uswds";
+
+import { USWDSIcon } from "src/components/USWDSIcon";
+
+export const selectForms = ({
```

Comment:
[suggestion] `selectRequiredForms`? `filterForRequiredForms`?

Reply:
right. How do we express that in the function name? `selectEitherRequiredOrNonRequiredForms`? (j/k)

Reply:
This feels pretty readable to me as is, howver I updated the name to `selectFormsByRequired`.